core/src/main/java/com/microsoft/applicationinsights/internal/schemav2/Envelope.java

```diff
@@ -139,8 +139,11 @@ public double getSampleRate() {
     /**
      * Sets the SampleRate property.
      */
-    public void setSampleRate(double value) {
-        this.sampleRate = value;
+    public void setSampleRate(Double value) {
```

Comment:
Sure, I can see and fix the code which tries to set this to null as you mentioned this file is autogenerated.